specutils/analysis/location.py

```diff
@@ -14,7 +14,7 @@
 __all__ = ['centroid']
 
 
-def centroid(spectrum, region):
+def centroid(spectrum, regions=None):
```

Comment:
This broke Jdaviz. Is this extra "s"  a typo? There is no deprecation period.

Example log: https://github.com/spacetelescope/jdaviz/runs/7290806527?check_suite_focus=true

Reply:
Gah, good call. See #963